core/CronArchive.php

```diff
@@ -1976,9 +1976,17 @@ private function getUrlsWithSegment($idSite, $period, $date)
                     return Request::ABORT;
                 }
 
-                $url = $request->getUrl();
-                $url = preg_replace('/([&?])date=[^&]*/', '$1date=' . $newDate, $url);
+                $urlBefore = $request->getUrl();
```

Comment:
@diosmosis does this also need to be done in https://github.com/matomo-org/matomo/blob/3.13.1/core/CronArchive.php#L1111 ?

Reply:
Yes, will add